iothub/device/src/Messaging/MessageSystemPropertyNames.cs

```diff
@@ -35,6 +35,8 @@ internal static class MessageSystemPropertyNames
 
         public const string CreationTimeUtc = "iothub-creation-time-utc";
 
+        public const string CreationTimeBatchUtc = "iothub-app-iothub-creation-time-utc";
```

Comment:
This is a supported D2C message system property which we weren't making available to be set